masterlist.yaml

```
@@ -14239,6 +14430,11 @@ plugins:
     clean:
       - crc: 0xD8B69F8C
         util: 'SSEEdit v4.0.3g'
+  - name: 'aMidianborn_Skyforge_Weapons.esp'
```

Since these mods are hosted on separate pages, they should be separated by a newline:

```suggestion
        util: 'SSEEdit v4.0.3g'

  - name: 'aMidianborn_Skyforge_Weapons.esp'
```